### PR TITLE
Migrate the StackPulse integration to Torq

### PR DESCRIPTION
**Pull request by hagaishapira**
### What does this PR do?

Convert the StackPulse integration into a Torq integration.

### Motivation

StackPulse is now Torq, we updated the integration with new relevant assets and copy.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: stackpulse/manifest.json [hunk 1]
```diff
@@ -1,33 +1,36 @@
 {
-  "display_name": "StackPulse",
-  "maintainer": "support@stackpulse.io",
+  "aliases": [
+    "/integrations/stackpulse/"
+   ],
+  "display_name": "Torq",
```
[the remark this answers is not in the document]
Comment (hagaishapira) [reply stackpulse/manifest.json]: @coignetp Done

**Comment by hagaishapira** [comment]
> Hi, we need to process the new integration logo on our side before being able to merge

Hi @coignetp Any update on the logo processing so we can go ahead and merge this PR?

File: stackpulse/assets/dashboards/torq_overview.json [hunk 2]
```diff
@@ -0,0 +1 @@
...lign":"center"}},{"id":2622851012825320,"layout":{"x":0,"y":15,"width":30,"height":42},"definition":{"type":"note","content":"### About Torq\n\n[Torq](https://torq.io) is a no-code automation platform for security and operations teams\n\n### Using this dashboard\n\nIn the dashboard you can track Torq originated events though a timeline and event stream. These events are produced by the Datadog Post Event step in torq workflows running in the [Torq App](https://app.torq.io). \n\nYou can use these events to keep track of workflow runs, send back status updates or even share enriched data collected by your workflows back to Datadog. ","background_color":"white","font_size":"14","text_align":"left","vertical_align":"top","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true}},{"id":3352112108208662,"layout":{"x":30,"y":0,"width":96,"height":8},"definition":{"type":"note","content":"# Torq Events","background_color":"vivid_blue","font_size":"14","text_align":"center","vertical_align":"center","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true}}],"template_variables":[],"layout_type":"free","is_read_only":false,"notify_list":[],"id":"4tm-v9h-xfh"}
```
Comment (alai97) [inline stackpulse/assets/dashboards/torq_overview.json:1]:
```suggestion
{"title":"Torq","description":"### Track all execution events posted from Torq workflows in one dashboard.\n\n### Use tags on events to separate events by environments, services, and applications. Or, build out dashboards. \n\n###For full visibility into your Torq workflow runs, go to [the Torq app](https//app.torq.io).","widgets":[{"id":2147516070905912,"layout":{"x":30,"y":19,"width":96,"height":50},"definition":{"title":"Torq Activity Stream","title_size":"16","title_align":"left","time":{"live_span":"1w"},"type":"event_stream","query":"sources:torq","tags_execution":"and","event_size":"s"}},{"id":6137349677227188,"layout":{"x":30,"y":8,"width":96,"height":9},"definition":{"title":"Torq Activity Timeline","title_size":"16","title_align":"left","time":{"live_span":"1w"},"type":"event_timeline","query":"sources:torq","tags_execution":"and"}},{"id":4444164082222724,"layout":{"x":5,"y":0,"width":18,"height":15},"definition":{"type":"image","url":"https://raw.githubusercontent.com/DataDog/integrations-extras/master/stackpulse/assets/logos/torq-black-horizontal.png","url_dark_theme":"https://raw.githubusercontent.com/DataDog/integrations-extras/master/stackpulse/assets/logos/torq-white-horizontal.png","sizing":"contain","margin":"sm","has_background":false,"has_border":false,"vertical_align":"center","horizontal_align":"center"}},{"id":2622851012825320,"layout":{"x":0,"y":15,"width":30,"height":42},"definition":{"type":"note","content":"### About Torq\n\n[Torq](https://torq.io) is a no-code automation platform for security and operations teams\n\n### Using this dashboard\n\nIn the dashboard, you can track Torq originated events though a timeline and event stream. These events are produced by the Datadog Post Event step in Torq workflows running in the [Torq App](https://app.torq.io). \n\nYou can use these events to keep track of workflow runs, send back status updates, or share enriched data collected by your workflows back to Datadog. ","background_color":"white","font_size":"14","text_align":"left","vertical_align":"top","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true}},{"id":3352112108208662,"layout":{"x":30,"y":0,"width":96,"height":8},"definition":{"type":"note","content":"# Torq Events","background_color":"vivid_blue","font_size":"14","text_align":"center","vertical_align":"center","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true}}],"template_variables":[],"layout_type":"free","is_read_only":false,"notify_list":[],"id":"4tm-v9h-xfh"}
```

**Review by alai97 — CHANGES_REQUESTED** [review]

**Comment by ChristineTChen** [comment]
Hi @hagaishapira,

I'm working on getting these changes synced to the Datadog UI. I am working with the design team to fix the torq logo to correspond with this integration's ID.

In the meantime, please see the above review from our documentations team.

**Comment by hagaishapira** [comment]
> Hi @hagaishapira,
> 
> I'm working on getting these changes synced to the Datadog UI. I am working with the design team to fix the torq logo to correspond with this integration's ID.
> 
> In the meantime, please see the above review from our documentations team.

Hi @ChristineTChen,
All done with applying the docs comments.

**Review by alai97 — APPROVED** [review]
Looks good for docs!

**Comment by ChristineTChen** [comment]
Hey @hagaishapira!

So I tried syncing these changes to the staging environment, and I think we're going to have to go a different route for this rebranding.

The display_name is currently affecting searching for `Torq` in the integration tile page in-app. For example, the Torq tile shows up under a `stackpulse` search.

Screen Shot 2021-11-08 at 2 09 42 PM

In this case, I believe the best path forward is to create a Torq tile (with torq integration-id and display_name in manifest.json). You can copy the contents of stackpulse to a `torq/` directory or create a new tile via `ddev create -t tile Torq` command and update the contents accordingly. The latter approach will be much safer in terms of ensuring the unique IDs and fields in manifest.json.

And for the existing stackpulse integration, we can hide the tile completely.

What do you think about this workaround?

**Comment by hagaishapira** [comment]
> Hey @hagaishapira!
> 
> So I tried syncing these changes to the staging environment, and I think we're going to have to go a different route for this rebranding.
> 
> The display_name is currently affecting searching for `Torq` in the integration tile page in-app. For example, the Torq tile shows up under a `stackpulse` search.
> 
> Screen Shot 2021-11-08 at 2 09 42 PM
> 
> In this case, I believe the best path forward is to create a Torq tile (with torq integration-id and display_name in manifest.json). You can copy the contents of stackpulse to a `torq/` directory or create a new tile via `ddev create -t tile Torq` command and update the contents accordingly. The latter approach will be much safer in terms of ensuring the unique IDs and fields in manifest.json.
> 
> And for the existing stackpulse integration, we can hide the tile completely.
> 
> What do you think about this workaround?

@ChristineTChen Sounds good to me, I'll make the changes and update back when they're in.

**Comment by hagaishapira** [comment]
Hi @ChristineTChen !
All done with having the Torq integration as a new tile. 
I left the StackPulse tile unchanged so if you can handle its hiding we would appreciate it. 

Hopefully we're good now 😃

File: torq/manifest.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "notification",
+    "collaboration",
+    "security"
+  ],
+  "type": "crawler",
+  "is_public": true,
+  "integration_id": "torq",
+  "assets": {
+    "dashboards": {
+      "StackPulse": "assets/dashboards/torq_overview.json"
```
Comment (ChristineTChen) [inline torq/manifest.json:30]:
```suggestion
      "Torq": "assets/dashboards/torq_overview.json"
```

**Review by ChristineTChen — CHANGES_REQUESTED** [review]
Let's update the dashboard key

File: torq/manifest.json (hunk 3 above)
Comment (hagaishapira) [reply torq/manifest.json:30]: Right, 2 mins...

**Comment by hagaishapira** [comment]
> Let's update the dashboard key

@ChristineTChen Done.

**Review by ChristineTChen — APPROVED** [review]

**Comment by hagaishapira** [comment]
@ChristineTChen @coignetp What is the next step in getting this merged?

**Pull request merged**